openproblems/tasks/_batch_integration/batch_integration_embed/checks.py

```diff
@@ -0,0 +1,12 @@
+def check_dataset(adata):
+    """Check that dataset output fits expected API."""
+    assert "batch" in adata.obs
+    assert "labels" in adata.obs
+    return True
+
+
+def check_method(adata):
+    """Check that method output fits expected API."""
+    # assert "template_output" in adata.obs
+    assert "distances" in adata.obsp
+    return True
```

Comment:
for `_embed` we should also assert that there is an `X_embed` output in `.obsm`. And this should be stated in the README.